django_file_form/forms.py

```diff
@@ -18,17 +18,16 @@ def __init__(self, *args, **kwargs):
         if s3_upload_dir is not None:
             self.add_hidden_field('s3_upload_dir', s3_upload_dir)
             self.add_s3_uploaded_files()
+            self.add_hidden_field('upload_url', reverse('s3_upload'))
+        else:
+            self.add_hidden_field('upload_url', reverse('tus_upload'))
 
         self.add_hidden_field('form_id', uuid.uuid4())
-        self.add_hidden_field('upload_url', self.get_upload_url())
         self.add_placeholder_inputs()
 
     def add_hidden_field(self, name, initial):
         self.fields[name] = CharField(widget=HiddenInput, initial=initial, required=False)
 
-    def get_upload_url(self):
```

Comment:
Removed because it is short but should restore if `get_upload_url` is an interface that needs to be exposed.